packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -119,12 +156,24 @@ const ImmunizationsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, close
     updateSingle('currentDose', currentDose);
   };
 
+  if (isLoading || vaccineState === undefined) {
+    return <span className={styles.vaccineLoader}>{`${t('loadingVaccine', 'Loading Vaccine')} ...`} </span>;
```

Comment:
Use the [Skeleton](https://react.carbondesignsystem.com/?path=/story/components-skeleton-skeletontext--default) or [Loader](https://react.carbondesignsystem.com/?path=/story/components-loading--default) from carbon react